service/notifications.go

```diff
@@ -0,0 +1,125 @@
+package service
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/RedHatInsights/sources-api-go/config"
+	"github.com/RedHatInsights/sources-api-go/kafka"
+	l "github.com/RedHatInsights/sources-api-go/logger"
+	m "github.com/RedHatInsights/sources-api-go/model"
+)
+
+var NotificationProducer Notifier
+
+type Notifier interface {
+	EmitAvailabilityStatusNotification(accountNumber string, emailNotificationInfo *m.EmailNotificationInfo) error
+}
+
+type AvailabilityStatusNotifier struct {
+}
+
+var (
+	application                = "sources"
+	notificationTopic          = config.Get().KafkaTopic("platform.notifications.ingress")
+	statusEventType            = "availability-status"
+	bundle                     = "console"
+	notificationMessageVersion = "v1.1.0"
+)
+
+type notificationPayload struct {
+}
+
+type notificationEvent struct {
+	Metadata string `json:"metadata"`
+	Payload  string `json:"payload"`
+}
+
+type notificationMetadata struct {
+}
```

Comment:
(minor): Probably worth adding some comments around these structs - I was a little confused at first but then I realized they're probably just here for the formatting of the JSON payload that they expect these fields even if they're empty.